microcosm_flask/namespaces.py

```diff
@@ -33,15 +33,14 @@ class Namespace(object):
 
     """
 
-    def __init__(
-        self,
-        subject,
-        object_=None,
-        path=None,
-        controller=None,
-        version=None,
-        enable_basic_auth=False
-    ):
+    def __init__(self,
+                 subject,
+                 object_=None,
+                 path=None,
+                 controller=None,
+                 version=None,
+                 enable_basic_auth=False,
+                 identifier_type="uuid"):
```

Comment:
The naming conventions that have an identifier type now *require* it as an argument. I couldn't find any examples where this was a breaking change. The default identifier type remains `uuid`.